config/manifests/bases/update-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |
+      [
+         {
+           "apiVersion": "updateservice.operator.openshift.io/v1",
+           "kind": "UpdateService",
+           "metadata": {
+             "name": "update-service-instance"
+           },
+           "spec": {
+             "replicas": 1,
+             "releases": "openshift-release-dev/ocp-release",
+             "graphDataImage": "quay.io/cincinnati/demo-graph-data:for-testing-only"
+           }
+         }
+       ]
+    capabilities: Basic Install
+    description: Creates and maintains an OpenShift Container Platform Update Service instance
+    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: update-service-operator.v1.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: UpdateService is the Schema for the updateservices API.
+      displayName: Update Service
+      kind: UpdateService
+      name: updateservices.updateservice.operator.openshift.io
+      version: v1
+  description: |
```

Comment:
I dunno if a trailing newline would cause any problems, but everyone the existing CSV is including a [`-` block chomping indicator][1] (so `description: |-`):

```console
$ git grep -B2 'in a cluster is appealing' origin/pr/93 | grep description:
origin/pr/93:config/manifests/bases/update-service-operator.clusterserviceversion.yaml-  description: |
origin/pr/93:config/olm-catalog/updateservice-operator/1.0.0/updateservice-operator.v1.0.0.clusterserviceversion.yaml-  description: |-
```

[1]: https://yaml.org/spec/1.2/spec.html#id2794534

Reply:
changed to `|-`